recipes/xeus-sqlite/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "xeus-sqlite" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/jupyter-xeus/xeus-sqlite/archive/{{ version }}.tar.gz
+  sha256: eacc8591fd26c009a407d88867796880b13ea27113931157e4abe52324ba4b74
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - xeus >=0.23.9,<0.24
+    - cppzmq >=4.4.1,<4.5.0
+    - zeromq >=4.2.5,<4.3.0
+    - xeus >=0.23.2,<0.24
+    - nlohmann_json >=3.2.0,<4.0
+    - xtl >=0.5.0,<0.7
+    - cpp-tabulate =1.0.0
+    - sqlitecpp >=3.0.0,<4
+    - sqlite >=3.30.1,<4
+  run:
+    # Even though cppzmq, xtl and nlohmann_json are header-only,
+    # they are used at runtime by the c++ interpreter.
+    - {{ pin_compatible('cppzmq', max_pin='x.x') }}
+    - {{ pin_compatible('xtl', max_pin='x.x') }}
+    - {{ pin_compatible('nlohmann_json', max_pin='x.x') }}
+
+test:
+  requires:
+    - jupyter_kernel_test
```

Comment:
```suggestion
```

Reply:
as you are just testing if those binaries are present, you do not need these lines